Menu/Menu.js

```diff
@@ -1,10 +1,12 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open')
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
```

Comment:
Good use of `document.querySelector` to target your menu and menu-button classes.